src/Expander.hs

```diff
@@ -587,14 +589,14 @@ initializeKernel = do
             p <- currentPhase
             es <- getExports exported
             modifyState $ over expanderModuleExports $ (<> es)
-            linkDecl dest (Export exported)
+            linkOneDecl dest (Export exported)
             nowValidAt pdest (SpecificPhase p)
         )
       , ( "meta"
         , \sc dest pdest stx -> do
             Stx _ _ (_ :: Syntax, subDecl) <- mustHaveEntries stx
-            subDest <- liftIO newDeclPtr
-            linkDecl dest (Meta subDest)
+            subDest <- liftIO newModBodyPtr
```

Comment:
Since the inside of a `meta` block isn't really a module body, but rather a location in which a group of declarations can be defined, I'm thinking of renaming `ModBody` to `DeclGroup`.

Reply:
I renamed it to `DeclTree` instead:

* `newModBodyPtr` and `linkModBody` are now `newDeclTreePtr` and `linkDeclTree`
* `expandOneModuleBodyNode` is now `expandOneDeclTreeNode`
* `_expanderCompletedModBody` is now `_expanderCompletedDeclTrees`
* `DeclDest` became `ModBodyDest` and is now `DeclTreeDest`
* `data ModuleBodyF ... = Done | Decl ...` became `data ModuleBodyF ... = NoDecls | Decl ... | Decls ...` and is now `data DeclTreeF ...  = DeclTreeLeaf | DeclTreeAtom ... | DeclTreeBranch ...`
* `SplitModuleBody` is now `SplitDeclTree`
* `ExpandDecl` and `ExpandMoreDecls` became `ExpandModBody` and `ExpandDependentModBody` and are now `ExpandDeclTree` and `ExpandDependentDeclTree`

but

* `forkExpandOneDecl` and `forkExpandDecls` are still named the same (I still want to rename them but I don't know to what)
* `getBody` is now `getDeclGroup`, not `getDeclTree`, because it returns a list and not a tree

Reply:
This all sounds reasonable to me.